src/test/java/com/hillel/items_exchange/util/UserDtoCreatingUtil.java

```diff
@@ -1,55 +1,66 @@
 package com.hillel.items_exchange.util;
 
+import com.hillel.items_exchange.dto.ChildDto;
+import com.hillel.items_exchange.dto.PhoneDto;
 import com.hillel.items_exchange.dto.UserDto;
 
-import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.Set;
 
 public class UserDtoCreatingUtil {
 
-    public static final long NOT_USER_ID = 100L;
     public static final String NEW_USERNAME = "newUsername123";
     public static final String NEW_EMAIL = "new.admin@gmail.com";
+    public static final LocalDateTime NEW_LAST_ONLINE_TIME = LocalDateTime.now();
     public static final String NEW_VALID_NAME_WITH_APOSTROPHE = "new'Name";
     public static final String NEW_VALID_NAME_WITH_HYPHEN_MINUS = "new-Name";
     public static final String NEW_INVALID_SHORT_NAME = "n";
     public static final String NEW_INVALID_TWO_WORDS_NAME = "new Name";
 
-    public static UserDto createUserDtoForUpdatingWithChangedEmailAndFirstNameApostrAndLastNameMinus() {
-        return getBuild(1L, "admin", NEW_EMAIL, Boolean.FALSE, NEW_VALID_NAME_WITH_APOSTROPHE,
-                NEW_VALID_NAME_WITH_HYPHEN_MINUS, "empty", LocalDate.of(2019, 1, 1));
+    public static UserDto createUserDtoForUpdatingWithChangedEmailAndFNameApAndLNameMinusWithoutChildrenOrPhones() {
+        return getBuild("admin", NEW_EMAIL, Boolean.FALSE, NEW_VALID_NAME_WITH_APOSTROPHE,
+                NEW_VALID_NAME_WITH_HYPHEN_MINUS, "empty", LocalDateTime.of(2019, 1, 1, 0, 0, 1),
+                Collections.emptySet(), Collections.emptySet());
```

Comment:
For now let it be with empty collections but soon we need replace them with real data.